data.sql

```diff
@@ -28,4 +28,51 @@ insert into animals (id,name,date_of_birth, escape_attempts, neutered, weight_kg
 values (10,'Blossom','1998-10-13',3,true,17);
 
 insert into animals (id,name,date_of_birth, escape_attempts, neutered, weight_kg)
-values (11,'Ditto','2022-05-14',4,true,22);
\ No newline at end of file
+values (11,'Ditto','2022-05-14',4,true,22);
+
+--- project three ---
+INSERT INTO owners (full_name, age) VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+INSERT INTO species (name) VALUES
+  ('Pokemon'),
+  ('Digimon');
+UPDATE animals
+SET species_id = 
+  CASE 
+    WHEN name LIKE '%mon' THEN (SELECT id FROM species WHERE name = 'Digimon')
+    ELSE (SELECT id FROM species WHERE name = 'Pokemon')
+  END;
+  UPDATE animals
+SET owner_id = owners.id
+FROM owners
+WHERE animals.name = 'Agumon' AND owners.full_name = 'Sam Smith';
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you make your update command something like the below to make your query query more optimized :blush:

```sql
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
```